framework/CodeInterfaces/PHISICS/phisicsdata.py

```diff
@@ -0,0 +1,961 @@
+    """
+    timeSteps.pop(0)
+    return timeSteps
+    
+  def getMrtauInstantTimeSteps(self):
+    """
+      Gets the time steps in the coupled MRTAU/INSTANT output.
+      @ In, None
+      @ Out, timeSteps, list
+    """
+    count = 0
+    timeSteps = []
+    with open(os.path.join(self.workingDir,self.mrtauOutputFileMPI[0]), 'r') as outfile:
+      for line in outfile:
+        if re.search(r'Material',line):
+          count = count + 1
+        if count == 1:
+          stringIsFloatNumber = self.isFloatNumber(line.split(','))
+          if stringIsFloatNumber is True:
+            timeSteps.append(line.split(',')[0])
+        if count > 1:
+          break
+    timeSteps = self.removeTheZeroTimeStep(timeSteps)
+    return timeSteps
+    
+  def getMrtauTimeSteps(self):
+    """
+      Gets the time steps in the MRTAU standalone output.
+      @ In, None
+      @ Out, timeSteps, list, list of time steps
+    """
+    timeSteps = []
+    with open(os.path.join(self.workingDir,self.mrtauOutputFileMPI[0]), 'r') as outfile:
+      for line in outfile:
+        stringIsFloatNumber = self.isFloatNumber(line.split(','))
+        if stringIsFloatNumber is True:
+          timeSteps.append(line.split(',')[0])
+    return timeSteps
+    
+  def getMrtauIsotopeList(self,mrtauCSVOutput):
+    """
+      Gets the isotope in the MRTAU standalone output.
+      @ In, mrtauCSVOutput, string, MRTAU file name
+      @ Out, None
+    """
+    self.isotopeListMrtau = []
+    with open(os.path.join(self.workingDir,self.mrtauOutputFileMPI[0]), 'r') as outfile:
+      for line in outfile:
+        if re.search(r'TIME',line):
+          line = re.sub(r'TIME\s+\(days\)',r'',line)
+          line = re.sub(r' ',r'',line)
+          line = re.sub(r'\n',r'',line)
+          self.isotopeListMrtau = filter(None,line.split(','))
+          break
+    
+  def getInstantTimeSteps(self,input):
+    """
+      Gets the time steps in the INSTANT output.
+      @ In, input, string, input file name
+      @ Out, timeSteps, list, list of time steps
+    """
+    count = 0
+    timeSteps = []
+    with open(os.path.join(self.workingDir,input), 'r') as outfile:
+      for line in outfile :
+        if re.search(r'PHISICS',line):
+          timeSteps.append(line.split(',')[line.split(',').index('Time') + 1])
+    return timeSteps
+
+  def reduceDigits(self,mrtauTimeSteps):
+    """
+      Reduces the number of digits in the MRTAU time steps.
+      @ In, mrtauTimeSteps, list
+      @ Out, timeSteps, list, list of time steps
+    """
+    timeSteps = []
+    for i in xrange (0, len(mrtauTimeSteps)):
+      timeSteps.append('%.6E' % (float(mrtauTimeSteps[i])))
+    return timeSteps
+  
+  def findUnits(self,line):
+    """
+      Returns the units in which the time steps are printed in the INSTANT output
+      @ In, line, string, line containing the units
+      @ Out, units, string
+    """ 
+    units = line.split(' ')[-3]
+    return units
```

Comment:
??? better way?

Reply:
yes there is. I am parsing the input to find that instead of looking at the output. Hence the method is implemented in the phisicsInterface. 
Same for the decay heat